docs/reference/mapping/types/sparse-vector.asciidoc

```diff
@@ -26,6 +26,19 @@ PUT my-index
 
 See <<semantic-search-elser, semantic search with ELSER>> for a complete example on adding documents to a `sparse_vector` mapped field using ELSER.
 
+[[sparse-vectors-params]]
+==== Parameters for `sparse_vector` fields
+
+The following parameters are accepted by `sparse_vector` fields:
+
+[horizontal]
+
+<<mapping-store,`store`>>::
+
+    Whether the field value should be stored and retrievable separately from
+    the <<mapping-source-field,`_source`>> field. Accepts `true` or `false`
+    (default).
+
```

Comment:
I know that it is a general mapping param, but would be nice to add a little explanation why users would want to set `store` to `true` (for example when _source is disabled and not synthetic source is available).

Reply:
++ - if not in the mapping doc, in a search guide

Reply:
I added some notes and links on the benefit.